plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/write/MapKeyValuesSchemaConverter.java

```diff
@@ -171,10 +175,9 @@ public static GroupType mapType(Repetition repetition, String alias, String mapA
     {
         //support projection only on key of a map
         if (valueType == null) {
-            return listWrapper(
+            return mapKvWrapper(
```

Comment:
Please avoid using shortcuts. `Kv` -> `KeyValue` ?

Reply:
Submitted https://github.com/trinodb/trino/pull/14105 to address this.